articles/how-crypto-mining-works/index.md

```diff
@@ -0,0 +1,75 @@
+
+### What Does Mining Do for the Cryptocurrency?  
+Mining can be thought of as the "processor" for transactions on the blockchain. [Bitcoin.org](bitcoin.org/en/how-it-works) describes the process of mining as "a distributed consensus system that is used to confirm pending transactions by including them in the block chain". Transactions are packaged into a block and then mined. This process of "mining" includes the transactions and verifies them in the chain. By continuing mining, these transactions are confirmed and become nearly unmodifiable by attacks on the chain. 
+
+In Bitcoin, every block is connected, as the previous block hash is used to generate a new block. If a hacker wanted to modify or "double-spend"
+
+### What Does A User Get Out of Mining?  
+While the chain and the cryptocurrency itself, benefits from users mining, users also benefit. A "miner" receives a reward for their work. This reward is generally a step amount per block and the fees of transactions that are included in the block. 
+
+The distributed mining power of the network also protects the blockchain from attacks, like double spending or any other transaction modifying attacks. In order for these attacks to be successful, the attacker would have to rehash the block they want to modify and make the longest valid chain of blocks (highest combined difficulty of the chain of blocks). This not only becomes harder the more blocks added after the attack block, but would likely require 51% of the total hashing power.
+
+### What is Proof-of-Work?  
+Bitcoin uses a proof-of-work algorithm. These algorithms provide a computational proof that can be easily checked. Most often, proof of work is implementing using various hashing algorithms with a target value. In order to successfully mine a block, a miner must generate a hash that is less than the target value with a certain difficulty value.
+
+In the case of Bitcoin, the target value is adjusted every 2016 blocks (about 2 weeks) in order to try and achieve 10 minute block times. By limiting the rate of blocks through a proof-of-work system, the coin supply is controlled and some attacks become impractical. 
+
+### How Does Bitcoin Mining Work?  
+In the case of mining Bitcoin, transactions are hashed and packaged. They are organized into a Merkle Tree. The following image from [Escape Velocity](https://chrispacia.wordpress.com/2013/09/02/bitcoin-mining-explained-like-youre-five-part-2-mechanics/) shows that transactions are initially hashed and then paired and hashed again. These hashes may be pair again and hash until you are at a single root. In this type of tree, the root is called the Merkle Root. 
+
+![](https://chrispacia.files.wordpress.com/2013/09/merkle-tree.jpg?w=300&h=267)
```

Comment:
Take a screenshot, include it in the folder directory, and then mention the source underneath the image. 

Including links makes the performance of the article and webpage dependent on the link inserted.